packages/client/modules/userDashboard/components/OrgBilling/OrgPlans.tsx

```diff
@@ -131,6 +139,30 @@ const OrgPlans = (props: Props) => {
   return (
     <>
       <StyledPanel label='Plans'>
+        {billingTier === 'enterprise' && (
```

Comment:
+1 the plans section doesn't feel like the right place for this, to me. The plans explain what is offered in each tier, whereas this is information about usage. 

I'm happy to still approve the PR if that's where you and @charlesmathias what to put it, but I'd suggest checking with @ackernaut first

Reply:
Sooner or later I’d like to create a usage section that is separate from the tiers and plans setion(s). Kind of like variants [we’ve explored here](https://www.figma.com/board/fdXXLw3wToZpVnpNvzoFl9/Org-Admin-%3E-Trial-Users?node-id=24-663&node-type=frame&t=YfZ6MBw2FTaFGWgl-11) 🔒 . It doesn’t have to happen in this PR.

I would like to see the usage section always at the top, regardless of how the other sections appear or are ordered below. If we need to add some special case banner to alert something at the very top, probably something that can be dismissed, we can always consider that later.